chrome-launcher/chrome-launcher.ts

```diff
@@ -285,6 +285,7 @@ export class Launcher {
             execSync(`taskkill /pid ${this.chrome.pid} /T /F`);
           } else {
             process.kill(-this.chrome.pid);
+            process.removeListener(_SIGINT, this.sigintListener);
```

Comment:
this brings up a decent point, should we only bind to signint if it's not windows then?